estate/models/estate_property.py

```diff
@@ -0,0 +1,35 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
```

Comment:
```suggestion
class EstateProperty(models.Model):
```
There is no big incidence here but we generally put the name of the file in CamelCase here.
It's usually good to be specific. `Property` is super generic and could be existing in other modules.